src/SmartComponents/Systems/SystemListAssets.test.js

```diff
@@ -34,7 +34,8 @@ describe('SystemListAssets.js', () => {
         it('Should call fetchApplicableSystemAdvisoriesApi with row id and limit: 10000', () => {
             expect(fetchApplicableSystemAdvisoriesApi).toHaveBeenCalledWith({
                 id: 'testId',
-                limit: 10000
+                limit: -1,
```

Comment:
@leSamo @mkholjuraev guys use pagination if possible, this way if there are 30k systems it will for sure time out on backend, the reasonable page size is 100 (maybe 1000 with fast apis)

Reply:
/views/systems/advisories with system id in POST body should be the best way how to do it IMHO, you don't need to deal with pagination and you will get all advisory ids for a single system